src/Mayaqua/TcpIp.c

```diff
@@ -4270,11 +4271,13 @@ BUF *DhcpModify(DHCP_MODIFY_OPTION *m, void *data, UINT size)
 		// Rewrite if anything changes. Do not rewrite if there is no change
 		ret_ok = true;
 
-		if (ret->Size < DHCP_MIN_SIZE)
+		// If src_size is greater than DHCP_MIN_SIZE, then use the src_size as minimum size of DHCP.
+		dhcp_min_size = max(src_size, DHCP_MIN_SIZE);

```

Comment:
"max" is not defined on linux, please see all windows builds failed

Reply:
Sorry, 
Macro MAX is defined. We can use it instead of max.
Do you need modified PR ?